test/common/cro_cr_success.go

```diff
@@ -0,0 +1,128 @@
+package common
+
+import (
+	goctx "context"
+	"encoding/json"
+	"fmt"
+	crov1 "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1"
+	croTypes "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1/types"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/resources/constants"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+)
+
+const (
+	requestNameSpace  = "redhat-rhmi-operator"
+	requestUrl        = "/apis/integreatly.org/v1alpha1"
+	openShiftProvider = "openshift"
+	externalProvider  = "aws"
+)
+
+func TestCROPostGresSuccessfulState(t *testing.T, ctx *TestingContext) {
+	postGresToCheck := []string{
+		fmt.Sprintf("%s%s", constants.CodeReadyPostgresPrefix, InstallationName),
+		fmt.Sprintf("%s%s", constants.ThreeScalePostgresPrefix, InstallationName),
+		fmt.Sprintf("%s%s", constants.RHSSOPostgresPrefix, InstallationName),
+		fmt.Sprintf("%s%s", constants.RHSSOUserProstgresPrefix, InstallationName),
+		fmt.Sprintf("%s%s", constants.UPSPostgresPrefix, InstallationName),
+		constants.AMQAuthServicePostgres,
+	}
```

Comment:
As part of the test case, we should also check for Fuse, which is not ready now so I thinking about which should be the best way to not forget this:

- add a TODO here, and create a follow-up task linked to the fuse task
- add a TODO here, and ensure that is part of the ac for the fuse task (my preferred)
- one of the previous without TODO

wdyt?

Reply:
I'll add a TODO here but yeah should be best to add it in also as part of the fuse task

Reply:
Requirement "added" to the task https://issues.redhat.com/browse/INTLY-3239

Reply:
Great thanks !